shared/src/components/SearchFilter/useBuildFilterOptions.tsx

```diff
@@ -659,6 +681,22 @@ const getOptionRoot = (
         operatorChangeable: false,
       }
       break
+    case 'productName':
+      rootOption = {
+        id: getRootIdWithPrefix(`name`),
+        type: 'string',
+        label: formatLabelWithScope(`Product Name`),
+        icon: getAttributeIcon('product'),
```

Comment:
This will give it the "text" icon which I think it more appropriate.

```suggestion
        icon: getAttributeIcon('productName', 'string'),
```